pubsub/pubsubhub.go

```diff
@@ -89,32 +76,24 @@ type connection struct {
 type PubSubHub struct {
 	liteMode   bool
 	sourceBase wsDataSource
-	sourceAux  wsDataSourceAux
 	wsHub      *WebsocketHub
 	state      *State
-	mempool    Mempool
 	params     *chaincfg.Params
+	invsMtx    sync.RWMutex
+	invs       *exptypes.MempoolInfo
 }
 
 // NewPubSubHub constructs a PubSubHub given a primary and auxiliary data
 // source. The primary data source is required, while the aux. source may be
 // nil, which indicates a "lite" mode of operation. The WebSocketHub is
 // automatically started.
-func NewPubSubHub(dataSource wsDataSource, auxDataSource wsDataSourceAux) *PubSubHub {
+func NewPubSubHub(dataSource wsDataSource, liteMode bool) (*PubSubHub, error) {
```

Comment:
We can get rid of the `liteMode` field and this input argument too.  If we need the aux DB in the future, we can a bring it back.

Reply:
removed.